keypirinha-steam/README.md

```diff
@@ -44,3 +48,4 @@ If needed, you can use the Keypirinha console to debug your Launchy configuratio
 - 2.2: Fix bug when a game doesn't have an icon
 - 2.3: Add support for Steam applications
 - 2.4: Fixed issue with Integer game names
+- 2.6: Updated various libraries top be able to parse appinfo.vdf v29 format
```

Comment:
nit: I technically forgot to update this, can you add

- 2.5: Update appinfo parsing library to support v28 format

Reply:
Added and pushed 👍